odl/discr/discr_ops.py

```diff
@@ -455,8 +455,10 @@ def _call(self, x, out=None):
         >>> f = par_deriv.domain.element(data)
         >>> par_div_f = par_deriv(f)
         >>> print(par_div_f)
-        [[0.0, 1.0, 2.0, 3.0, 4.0],
-         [0.0, 1.0, 2.0, 3.0, 4.0]]
+        [
```

Comment:
I remember a comment like "this is ugly, eh?" but couldn't locate it. Did the quick fix to the doctest instead, agree, the old version was more readable.